python-sdk/src/astro/databases/base.py

```diff
@@ -683,7 +683,7 @@ def export_table_to_file(
     # Schema Management
     # ---------------------------------------------------------
 
-    def create_schema_if_needed(self, schema: str | None) -> None:
+    def create_schema_if_needed(self, schema: str | None, location: str | None = None) -> None:
```

Comment:
In some parts of the code, we're using region, and here we're using location. Would it make sense to be consistent and only use region? Are there any circumstances when we want to use zone?

Reply:
@tatiana Good point, I think we should replace it with the region will avoid confusion with the file's location attribute.